examples/server/server.cpp

```diff
@@ -3388,6 +3388,24 @@ int main(int argc, char ** argv) {
     // struct that contains llama context and inference
     server_context ctx_server;
 
+    shutdown_handler = [&](int) {
```

Comment:
I don't get why this whole code block must be this higher up. Can't it be move to just right before `ctx_server.queue_tasks.start_loop();` ?

Reply:
Hi @ngxson , I think yes.